Assets/Mirror/Runtime/Messages.cs

```diff
@@ -171,6 +171,36 @@ public void Serialize(NetworkWriter writer)
         }
     }
 
+    public struct AuthRequestMessage : IMessageBase
+    {
+        public byte[] value;
+
+        public void Deserialize(NetworkReader reader)
+        {
+            value = reader.ReadBytesAndSize();
+        }
+
+        public void Serialize(NetworkWriter writer)
+        {
+            writer.WriteBytesAndSize(value);
+        }
+    }
+
+    public struct AuthResponseMessage : IMessageBase
```

Comment:
do we need a built in response message?   the user could just add their own message if they wanted no?